.github/workflows/php-cs-fixer.yml

```diff
@@ -18,10 +18,7 @@ jobs:
 
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
-        with:
-          ref: ${{ github.head_ref }}
-          token: ${{ secrets.PHP_CS_FIXER }}
+        uses: actions/checkout@v3
```

Comment:
I made the above changes to allow the workflow to run on my fork without failing.

Am open to creating a separate PR for this